### DISCUSSION TITLE
Budibase September Release

### DISCUSSION THREAD

**Discussion by shogunpurple** [Budibase/budibase#2926] [Announcements]
# September release notes - Cloudy, with a chance of awesomeness

The September release introduces Budibase Cloud, making it invitingly easy to try Budibase and build internal tools in minutes. This is our largest and most significant release to date. 

### Budibase Cloud is now in beta

![image](https://user-images.githubusercontent.com/11256663/136466808-20dbd4e0-a122-4609-989c-d5fd8ab1205c.png)

Before Budibase Cloud, it wasn't easy to try Budibase if you were unfamiliar with Docker, Kubernetes, or Digital Ocean. With Budibase Cloud, you can experience the power, sleekness, and simplicity of Budibase in just a few seconds. It is also the beginning of a momentous drive to make Budibase more accessible and intuitive, and offer a best-in-class experience. 

Before jumping in, there are a few points we would like you to know:

* Budibase Cloud is in beta.
* Budibase Cloud is free (we will add paid plans in the future)
* Each new account can build up to 4 applications, run 1000 automation operations, and store 1000 records per app. \

Try Budibase today. Setup takes seconds, and it's free. And if you do enjoy your experience, please tell a friend or two about it. \
 https://account.budibase.app/register?utm_source=github-discussions&utm_medium=release-notes&utm_campaign=september-release-notes

### Theming

We obsess over design at Budibase, and we understand and actively promote the importance of creating internal tools people enjoy using. Theming gives you the creative flexibility to tailor your internal tools to suit your brand. Currently, you can change the overall theme, accent color (button, links), button shape, button hover color, navigation background color, navigation font color). 

![image](https://user-images.githubusercontent.com/11256663/136466842-112993d0-8904-428d-85a7-9c66d1f5e01f.png)

### Desktop, tablet, mobile view

Unlike other platforms, Budibase apps work on mobile, tablet, and desktop. The new device toggle lets you change the design screen dimensions with just a click, making it incredibly easy to design for different devices.

![image](https://user-images.githubusercontent.com/11256663/136466874-295d56f0-3e62-4622-867c-fde1c3780c55.png)

### Drag and drop screen components

A popular request amongst the community, you can now add components to your design screen then drag and drop them into the position of your choice.

![image](https://user-images.githubusercontent.com/11256663/136466892-87378b06-2a88-49f0-ad53-c5cf8c8ec69f.png)

### Automations redesign

We redesigned the automations UI to make it easier for users to create new automations.

![image](https://user-images.githubusercontent.com/11256663/136466921-3ebdd016-4884-4426-adb0-1e43e50b38df.png)

### Automation test functionality

The new testing functionality helps you test your integration and receive feedback regards to your triggers and actions.

### New query action in automations

The new query automation action allows you to get rows from a table the same way you would use a data provider.

### Search records within your data

Within the data section of the builder, you can now search for records. This feature is only available for internal tables, PostgreSQL, MySQL.

**Comment by berma83** [comment]
My app is running much faster after this update.  :rocket::rocket::rocket:

Thank you!

**Reply by joebudi** [reply]
Thanks for the feedback @berma83

**Reply by tobehn** [reply]
Congrats @BB-Team..

I wish you a lot of success with the start up of your cloud service.

You have created something valuable which gets better and better even in the tiniest corners.

I am looking forward to check out all new features and find all the fine tuned enhancements under the hood that make my total user experience flawless.

**Reply by mjashanks** [reply]
Thanks @tobehn! 

And thanks to you and everyone for your feedback and contributions to this community - it is incredibly important for us.

**Comment by KernOil** [comment]
Hi, Team Great work on the new Update! you guys are crushing it !!
